### DISCUSSION TITLE
[9 Settembre 2022] - Pipeline

### DISCUSSION THREAD

**Discussion by notedo** [CS-Swap/Architettura-degli-Elaboratori#18] [Pipeline]
### Traccia Esercizio
<img width="816" alt="asdasd" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/c8328a69-9f39-4521-9c64-2fcb76c8f92c">
<img width="784" alt="Scrasdasdeenasdasd" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/a8579098-9efb-4f29-b8a2-b0d9f23995af">
<img width="867" alt="Seg" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/82aa71bf-2616-44d2-977f-bda35f004eb3">

**Comment by thegeek-sys** [comment]
# Senza forwarding
<img width="1083" alt="Screenshot 2024-06-03 alle 21 56 20" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/d2fd113a-ae7e-42d0-9c85-19ed5f33d0bc">

# Con  forwarding
<img width="1083" alt="Screenshot 2024-06-03 alle 21 57 31" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/179e41cb-9e17-4488-a4b6-928d6df11e39">

# Control hazard e data hazard
<img width="239" alt="Screenshot 2024-06-03 alle 21 58 23" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/853520be-82c2-4a5c-bb47-907b9ec8d876">

# Fasi delle istruzioni al 14° ciclo di clock
<img width="87" alt="Screenshot 2024-06-03 alle 21 59 02" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/7ec6919a-3291-43c3-a216-449d186b5f52">
